src/Sarif/CodeGenHints.json

```diff
@@ -512,13 +657,25 @@
   "Run.OriginalUriBaseIds": [
     {
       "kind": "DictionaryHint"
-    },
+    }
```

Comment:
>} [](start = 4, length = 1)

I assume you omitted the `AttributeHint` because with the new "chained" design for `originalUriBaseIds`, the dictionary entries are no longer necessarily URIs.